src/command_modules/azure-cli-vm/tests/test_vm_commands.py

```diff
@@ -287,6 +287,55 @@ def test_windows_vm_license_type(self, resource_group):
         ])
 
 
+class VMCustomImageTest(ScenarioTest):
+
+    @ResourceGroupPreparer()
+    def test_custom_image(self, resource_group, resource_group_location):
+        # this test should be recorded using accounts "@azuresdkteam.onmicrosoft.com", as it uses pre-made generalized vms
+        prepared_vm_unmanaged = '/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/sdk-test/providers/Microsoft.Compute/virtualMachines/sdk-test-um'
```

Comment:
@troydai, I can move the pre-made vms to the `node` subscription so it can run under live, but let me know if we are switching out of the `node` soon.